src/main/kotlin/receiver/Receiver.kt

```diff
@@ -0,0 +1,95 @@
+package receiver
+
+import KeyVersionPair
+import NodeId
+import com.fasterxml.jackson.databind.ObjectMapper
+import cache.distributed.IDistributedCache
+import io.javalin.Javalin
+import io.javalin.http.HttpCode
+import org.eclipse.jetty.http.HttpStatus
+
+/**
+ * A concrete receiver that accepts requests over HTTP.
+ */
+class Receiver(private val nodeId: NodeId, private val distributedCache: IDistributedCache) : IReceiver {
+
+    /**
+     * The Javalin server used to route HTTP requests to handlers
+     */
+    private val app: Javalin = Javalin.create()
+
+    /**
+     * The ObjectMapper used to decode JSON data
+     */
+    private val mapper: ObjectMapper = ObjectMapper()
+
+    init {
+        /* Handle store requests */
+        app.post("/store/{key}/{version}") { ctx ->
```

Comment:
We should discuss how we want to format our REST endpoints and probably add an API version in case we ever want to change things. Maybe `/v1/cache/{key}/{version}` or even `/v1/blobs/{key}/{version}`? Regardless, fetch and store should probably use same endpoint